cristin-biobank/src/main/java/no/unit/nva/biobank/model/cristin/CristinAssocProjectForBiobank.java

```diff
@@ -0,0 +1,48 @@
+package no.unit.nva.biobank.model.cristin;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.net.URI;
+import java.util.Map;
+
+public class CristinAssocProjectForBiobank {
+
+    public static final String CRISTIN_PROJECT_IDENTIFICATOR = "cristin_project_id";
+    public static final String CRISTIN_PROJECT_ID = "url";
+    public static final String CRISTIN_PROJECT_TITLE = "title";
+
+    @JsonProperty(CRISTIN_PROJECT_ID)
+    private final URI cristinProjectId;
+    @JsonProperty(CRISTIN_PROJECT_IDENTIFICATOR)
+    private final String cristinProjectIdentificator;
+
+    @JsonProperty(CRISTIN_PROJECT_TITLE)
+    private final Map<String, String> title;
+
+
+    /**
+     * Constructor
+     * @param cristinProjectId - URI
+     * @param cristinProjectIdentificator - code
+     * @param title - name of the project
+     */
+    public CristinAssocProjectForBiobank(URI cristinProjectId,
```

Comment:
We might want to consider for later annotating with JsonCreator and adding the JsonProperty here as well, but we dont need to do that yet until the need arrives.

Also for later, consider creating the toString, equals and hashcode if we need them

Reply:
Is also relevant other places, but not very important yet